### PR TITLE
IBX-2500: Fixed number of iterations in `normalize-paths` command

### PR DESCRIPTION
**Pull request by barw4**
| Question                                  | Answer
| ---------------------------------------- | ------------------
| **JIRA issue**                          | [IBX-2500](https://issues.ibexa.co/browse/IBX-2500)
| **Type**                                   | bug
| **Target eZ Platform version** | `v2.5`
| **BC breaks**                          | no

Previously it could occur, that when having multiple translations and versions, iterations could be finished before the command had processed all images.

#### Checklist:
- [x] Provided PR description.
- [x] Tested the solution manually.
- [ ] Provided automated test coverage.
- [x] Checked that target branch is set correctly (master for features, the oldest supported for bugs).
- [x] Ran PHP CS Fixer for new PHP code (use `$ composer fix-cs`).
- [x] Asked for a review (ping `@ezsystems/engineering-team`).

### CODE REVIEW

**Review by alongosz — CHANGES_REQUESTED** [review]
Can't dig deeper atm, based on what I see in the PR & entire class definition:
`$this->imageGateway` in the command is of type `eZ\Publish\Core\FieldType\Image\ImageStorage\Gateway`, yet you're changing `eZ\Publish\Core\FieldType\Image\ImageStorage\Gateway\DoctrineStorage` only. Something doesn't add up here. Was `countDistinctImages` not defined in the abstract Gateway?

**Comment by barw4** [comment]
> Was countDistinctImages not defined in the abstract Gateway?

@alongosz you are correct, it wasn't.

**Comment by alongosz** [comment]
@barw4 then this needs to be fixed. Now it relies on a polymorphic call which should be visible in PHPStorm via static analysis.

**Comment by barw4** [comment]
@alongosz done

**Review by mateuszbieniek — APPROVED** [review]

**Comment by mateuszbieniek** [comment]
Looks good to me

**Review by alongosz — APPROVED** [review]

**Comment by alongosz** [comment]
@mateuszbieniek once it gets 2nd maintainer approve it can go to QA.

**Review by webhdx — APPROVED** [review]

**Review by Steveb-p — APPROVED** [review]

**Review by damianz5 — APPROVED** [review]

**Review by micszo — APPROVED** [review]
QA Approved on eZ Platform EE v2.5.27 with diff.

**Comment by barw4** [comment]
Merged into:
`ezsystems/ezplatform-kernel/1.3`: https://github.com/ezsystems/ezplatform-kernel/commit/ca232009d90f0a4fa723452e303b3aa7812c1926
`ibexa/core/main`: https://github.com/ibexa/core/pull/95

**Pull request merged**
